unified/install.sh

```diff
@@ -88,6 +95,22 @@ if [ -z "$prefix" ]; then
     fi
 fi
 
+if [ -z "$python3" ]; then
+    if $nonroot; then
+        python3=~/scylladb/python3/bin/python3
+    else
+        python3=/opt/scylladb/python3/bin/python3
```

Comment:
should use "$prefix/" as a prefix, so users can override the default for all files with just --prefix.

Reply:
good catch! fixed.